examples/inverse/plot_morph_VolSourceEstimate.py

```diff
@@ -0,0 +1,282 @@
+#
+# License: BSD (3-clause)
+
+from os import makedirs, path
+
+import matplotlib.pylab as plt
+import mne
+import nibabel as nib
+import numpy as np
+from dipy.align import imaffine, imwarp, reslice
+from mne.beamformer import make_lcmv, apply_lcmv
+from mne.datasets import sample
+from mne.externals.h5io import write_hdf5, read_hdf5
+from nilearn.image import index_img
+from nilearn.plotting import plot_anat
+
+print(__doc__)
+
+
+###############################################################################
+# from :ref:`LCMV beamformer inverse example
+# <sphx_glr_auto_examples_inverse_plot_lcmv_beamformer_volume.py>`. Note, that
+# in order to save the data fname needs to be set
+
+def compute_lcmv_example_data(data_path, fname=None):
+    raw_fname = data_path + '/MEG/sample/sample_audvis_raw.fif'
+    event_fname = data_path + '/MEG/sample/sample_audvis_raw-eve.fif'
+    fname_fwd = data_path + '/MEG/sample/sample_audvis-meg-vol-7-fwd.fif'
+    # Get epochs
+    event_id, tmin, tmax = [1, 2], -0.2, 0.5
+
+    # Setup for reading the raw data
+    raw = mne.io.read_raw_fif(raw_fname, preload=True)
+    raw.info['bads'] = ['MEG 2443', 'EEG 053']  # 2 bad channels
+    events = mne.read_events(event_fname)
+
+    # Set up pick list: gradiometers and magnetometers, excluding bad channels
+    picks = mne.pick_types(raw.info, meg=True, eeg=False, stim=True, eog=True,
+                           exclude='bads')
+
+    # Pick the channels of interest
+    raw.pick_channels([raw.ch_names[pick] for pick in picks])
+
+    # Re-normalize our empty-room projectors, so they are fine after
+    # subselection
+    raw.info.normalize_proj()
+    # Read epochs
+    proj = False  # already applied
+    epochs = mne.Epochs(raw, events, event_id, tmin, tmax,
+                        baseline=(None, 0), preload=True, proj=proj,
+                        reject=dict(grad=4000e-13, mag=4e-12, eog=150e-6))
+    evoked = epochs.average()
+
+    # Read regularized noise covariance and compute regularized data covariance
+    noise_cov = mne.compute_covariance(epochs, tmin=tmin, tmax=0,
+                                       method='shrunk')
+    data_cov = mne.compute_covariance(epochs, tmin=0.04, tmax=0.15,
+                                      method='shrunk')
+    # Read forward model
+    forward = mne.read_forward_solution(fname_fwd)
+
+    # Compute weights of free orientation (vector) beamformer with weight
+    # normalization (neural activity index, NAI). Providing a noise covariance
+    # matrix enables whitening of the data and forward solution. Source
+    # orientation is optimized by setting pick_ori to 'max-power'.
+    # weight_norm can also be set to 'unit-noise-gain'. Source orientation can
+    # also be 'normal' (but only when using a surface-based source space) or
+    # None, which computes a vector beamfomer. Note, however, that not all
+    # combinations of orientation selection and weight normalization are
+    # implemented yet.
```

Comment:
let's remove this comment -- try to minimize the number of lines spent on the "we are doing beamforming" to better accentuate the point of the example, "we are morphing volume estimates"

Reply:
might be from the example that I copied... I'll remove / change